src/port1.0/portconfigure.tcl

```diff
@@ -1367,27 +1367,23 @@ proc portconfigure::configure_main {args} {
     foreach flags {cflags cppflags cxxflags objcflags objcxxflags ldflags fflags f90flags fcflags} {
         global configure.${flags} configure.universal_${flags}
     }
-
-    if {[tbool use_autoreconf]} {
-        if {[catch {command_exec autoreconf} result]} {
-            return -code error "[format [msgcat::mc "%s failure: %s"] autoreconf $result]"
-        }
-    }
-
-    if {[tbool use_automake]} {
-        if {[catch {command_exec automake} result]} {
-            return -code error "[format [msgcat::mc "%s failure: %s"] automake $result]"
-        }
-    }
-
-    if {[tbool use_autoconf]} {
-        if {[catch {command_exec autoconf} result]} {
-            return -code error "[format [msgcat::mc "%s failure: %s"] autoconf $result]"
+    # Export flags into destroot and build
+    foreach phase {destroot build} {
+        parse_environment ${phase}
```

Comment:
I could be mistaken, but I believe `parse_environment` is already called [by default](https://github.com/macports/macports-base/blob/6f8aca0267707bd4d8e122148e3f88af57029a73/src/port1.0/portutil.tcl#L415).